scripts/platoon_launcher

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import os
+import time
+import argparse
+import subprocess
+
+
+def parse_arguments():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('experiment_name', help='The name of your experiment. The launcher will expect to find the files <experiment_name>_controller.py and <experiment_name>_worker.py.')
+    parser.add_argument('gpu_list', nargs='+', type=int, help='The list of GPU ids the script will use. 1 GPU id = 1 worker launched.')
+    parser.add_argument('-c', '--controller-args', required=False, help='The arguments that will be passed to your controller. Those should be put in quotes. (Ex: -c=\"--sync_rule EASGD\")')
+    parser.add_argument('-w', '--workers-args', required=False, help='The arguments that will be passed to your workers.  Those should be put in quotes. (Ex: -w=\"learning_rate=0.1\")')
+
+    return parser.parse_args()
+
+
+def launch_process(logs_folder, experiment_name, args, worker_id=None):
+    if worker_id is None:
+        worker_id = ""
+        device = "cpu"
+        process_type = "controller"
+    else:
+        process_type = "worker"
+        device = "gpu{}".format(worker_id)
```

Comment:
It would be nice to have an option for it to be `cuda*` for it to be usable with the `gpuarray` back-end.